### PR TITLE
deny exec error and enforce deprecation issues resolved

### PR DESCRIPTION
**Pull request by nayanballa08**
Error and Success showing Images attached under guide/static/img.

Updated yaml file under /scenarios/kyverno*/ to fix the error.

### CODE REVIEW

**Comment by nayanballa08** [comment]
Please help review and merge to the master as applicable.

As part of my learning I came across this issue.

**Comment by madhuakula** [comment]
Hello @nayanballa08 

Thanks for the fix, I think the newer version of Kyverno 1.10.0 has these changes. Let me validate and merge the changes. 
https://kyverno.io/policies/other/block-pod-exec-by-namespace/block-pod-exec-by-namespace/

I was away for a while, apologies for the delays

File: guide/static/img/deny-exec-error.png
Comment (madhuakula) [inline guide/static/img/deny-exec-error.png:1]: can we remove this file as its part of docs, I will try updating this with same styling.

File: guide/static/img/deny-exec-success.png
Comment (madhuakula) [inline guide/static/img/deny-exec-success.png:1]: can we remove this file as its part of docs, I will try updating this with same styling.

**Comment by nayanballa08** [comment]
Hi @madhuakula thanks for taking time to review, as per your comments I have deleted the png files.
FYI I started posting youtube videos on walkthroughs about Kubernetes Goat scenarios 
This is the channel: https://www.youtube.com/watch?v=wgY-rfcauO4&list=PLk5LcDGteX9pjZim10J1xII9hf5KQjC4h
Aiming to cover all the scenarios as we go.

**Comment by madhuakula** [comment]
Hey @nayanballa08 

Thanks for the PR. Btw, those videos are wonderful for sharing with the folks. Thanks for making them with your explanations. 

Looking forward for more content, contributions and suggestion.

**Pull request merged**